calculator.go

```diff
@@ -1,13 +1,89 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"fmt"
+	"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(nums ...float64) float64 {
+	var result float64 = nums[0]
```

Comment:
A minor point of idiomatic Go is that the `var` keyword is usually only used when we're happy for the variable to receive its default (zero) value. If we want to declare and then assign something non-zero to it, we often do that in one step using the short declaration form `:=`.

Reply:
Addressed in a9e147c.

Question: should I be doing this [here](https://github.com/bitfieldconsulting/ftl-fundamentals-pmamberti/blob/a9e147c9f0cdd56adc399fd26c007b938c5a52a9/calculator.go#L72) as well? 
In my head I'm happy for these to be 0, but I may be missing something 🤔

Reply:
Yes, that's fine—we don't care what their value is, since it will be overwritten by `Scanf` anyway.